test/test_04_03.py

```diff
@@ -0,0 +1,13 @@
+from hm_1.tasks.task_04_03 import my_precious_logger
+
+
+def test_my_precious_logger_returns_error(capsys):
+    my_precious_logger("error: file not found")
+    out, err = capsys.readouterr()
+    assert (out, err) == ("", "error: file not found")
```

Comment:
I'd prefer to make it more vertical.

```suggestion
    assert out == ""
    assert err == "error: file not found"
```